### PR TITLE
chore(dev): fix chromedriver crash during tests in devcontainer

### PR DESCRIPTION
**Pull request by NGPixel**
Fixes chromedriver crashing during tests by setting `1777` permissions to `/dev/shm`

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -74,9 +74,9 @@
 		"mrmlnc.vscode-duplicate",
 		"ms-azuretools.vscode-docker",
 		"ms-python.python",
-		"ms-python.vscode-pylance",
+	    	"ms-python.vscode-pylance",
```
Comment (rjsparks) [inline .devcontainer/devcontainer.json:77]: Is this intended?

File: .vscode/extensions.json [hunk 2]
```diff
@@ -1,6 +1,3 @@
 {
-  "recommendations": [
-    "arcanis.vscode-zipfs",
-    "dbaeumer.vscode-eslint"
-  ]
+  "recommendations": []
```
Comment (rjsparks) [inline .vscode/extensions.json:2]: This also seems unrelated to this particular problem?

**Review by rjsparks — CHANGES_REQUESTED** [review]

File: .vscode/extensions.json (hunk 2 above)
Comment (NGPixel) [reply .vscode/extensions.json:2]: It's unrelated but it's to silence some popups when opening vscode. I can make a separate PR if you prefer, but it's quite a tiny change with no impact.
